public/locales/en/account.json

```diff
@@ -0,0 +1,18 @@
+{
+  "pageTitle": "Account",
+  "tabs": {
+    "myData": "My data",
+    "notifications": "Notifications",
+    "accountInformation": "Account information",
+    "apiToken": "API token"
```

Comment:
```suggestion
    "myData": "My Data",
    "notifications": "Notifications",
    "accountInformation": "Account Information",
    "apiToken": "API Token"
```

Do we want Title Case or not? 🤔 Do we have any docs on which to use? I checked the Style Guide but I didn't see anything. In JSF we used Title Case for these:

![Screenshot 2024-09-06 at 9 26 18 AM](https://github.com/user-attachments/assets/12b205c4-55e2-49f7-b2bc-c5f2ddab4032)

Reply:
Hi @pdurbin, yes indeed !! I will commit your suggestion, many thanks!